lib/archethic/shared_secrets/mem_tables_loader.ex

```diff
@@ -86,7 +86,7 @@ defmodule Archethic.SharedSecrets.MemTablesLoader do
 
   def load_transaction(%Transaction{
         address: address,
-        type: :origin_shared_secrets,
+        type: :origin,
         data: %TransactionData{content: content}
       }) do
     content
```

Comment:
There is also a bug here, as now we have only one key and a certificate in the content of the transaction, we should deserialize it as in the mining part to get the public key.